Sprint-1/fix/median.test.js

```diff
@@ -8,7 +8,7 @@ const calculateMedian = require("./median.js");
 
 describe("calculateMedian", () => {
   test("returns the median for odd length array", () => {
-    expect(calculateMedian([1, 2, 3])).toEqual(2);
+   // expect(calculateMedian([1, 2, 3])).toEqual(2);
```

Comment:
**Never** delete a test which is correct just because it fails.  The median of `[1, 2, 3]` is indeed `2`.